### PR TITLE
Add translation support via `morphi` library

### LINKED ISSUE #41: Add Multilingual Support

### PR DESCRIPTION
**Pull request by tjlevel12**
Includes initial Spanish translations.

Fixes #41

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,11 +56,16 @@
     extras_require={'develop': develop_requires},
     zip_safe=False,
     include_package_data=True,
+    setup_requires=[
+        'Babel'
+    ],
     install_requires=[
         'BlazeUtils',
         'FormEncode',
+        'Flask-Script',
```
[the remark this answers is not in the document]
Comment (tjlevel12) [reply setup.py]: The [test app](https://github.com/level12/webgrid/blob/fd7837ab4b161fd3a3f6974221d0adc7d396125a/webgrid_ta/manage.py#L7) uses it. I suppose it doesn't have to be in `install_requires`; I could add a 'testapp' extra, if you'd prefer.

[the remark this answers is not in the document]
Comment (rsyring) [reply setup.py]: Looks like test requirements should go in:

https://github.com/level12/webgrid/blob/68dc1ad69518526a420005e5e0556875fa85b3af/setup.py#L11

https://github.com/level12/webgrid/blob/68dc1ad69518526a420005e5e0556875fa85b3af/tox.ini#L12

[the remark this answers is not in the document]
Comment (tjlevel12) [reply setup.py]: The test app isn't run by the test suite, though, it's run via `manage.py serve` (or `manage.py runserver`, I'm not really clear as to the difference between those two).

I'm fine moving it to `develop_requires` if you think that's the best place for it. You don't really *need* the test app to develop webgrid, it just gives you a demo of the functionality.

[the remark this answers is not in the document]
Comment (guruofgentoo) [reply setup.py]: `develop_requires` will be fine here. I'm not concerned about having a few extra dependencies there to develop, run tests, and run the test app. I -am- concerned about having it in the install requirements.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Babel configuration
+
+[extract_messages]
+input_dirs = webgrid
+mapping_file = webgrid/i18n/babel.cfg
+#keywords = _ ngettext
```
Comment (guruofgentoo) [inline setup.cfg]: Maybe pull this line if it isn't needed.

**Comment by tjlevel12** [comment]
@guruofgentoo I've pulled out `morphi` into an optional dependency (via `[i18n]`).

I didn't want to handle the template translations in the code, as that seemed like a good way to not callout any future strings. Instead, I added some code to each template which defines the underscore function if it doesn't already exist.

Overall, there's way more duplicated code here than I would like, but I don't see a good way around it.

**Comment by guruofgentoo** [comment]
WRT backward compatibility, when I load this in one of our apps, I am seeing the following in the browser console:
```
jquery.multiple.select.js:441 Uncaught ReferenceError: _ is not defined
    at jquery.multiple.select.js:441
    at jquery.multiple.select.js:466
```

**Pull request merged**